components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -11752,6 +11771,7 @@ public static boolean isCrossTenantSubscriptionsEnabled() {
         return false;
     }
 
+
```

Comment:
Shall we remove this extra line?

Reply:
Fixed in 3e4283c7d975e6e9d7c32feac5a0f5f1914359ed